app/models/importers/hmis_six_one_one/base.rb

```diff
@@ -437,6 +450,104 @@ def clean_source_file destination_path:, read_from:, klass:
       write_to.close
     end
 
+    # We sometimes see very odd dates, this will attempt to make them sane.
+    # Since most dates should be not too far in the future, we'll check for anything less
+    # Than a year out
+    private def fix_date_format(string)
+      return unless string
+      # Ruby handles yyyy-m-d just fine, so we'll allow that even though it doesn't match the spec
+      return string if /\d{4}-\d{1,2}-\d{1,2}/.match?(string)
+
+      # Sometimes dates come in mm-dd-yyyy and Ruby Date really doesn't like that.
+      if /\d{1,2}-\d{1,2}-\d{4}/.match?(string)
+        month, day, year = string.split('-')
+        return "#{year}-#{month}-#{day}"
+      # Sometimes dates come in mm/dd/yyyy
+      # Sometimes times come in mm/dd/yyyy hh:mm
```

Comment:
Since we don't recognize this below (since this is for dates), remove this line, here?
(And adjust the other copies as appropriate)